BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee

```diff
@@ -37,11 +37,12 @@ do ->
       if !@templates[language][templateName]
         templateLoader = this
         filename = appRouter.generateUrl('loadUnderscoreTemplate')
-        jQuery.get filename, {
+        jqXHR = jQuery.get filename, {
```

Comment:
i think it's more explicit to use ajax property with statuscode method, you have an example in SubmitAdmin.coffee